composer.json

```diff
@@ -37,10 +38,16 @@
 		"szepeviktor/phpstan-wordpress": "^1.3",
 		"wp-cli/wp-cli": "^2.9"
 	},
+	"repositories": [
+		{
+			"type": "vcs",
+			"url": "https://github.com/infinum/eightshift-forms-utils.git"
+		}
+	],
 	"require": {
 		"php": "^7.4 || >=8.0",
 		"erusev/parsedown": "^1.7.4",
-		"infinum/eightshift-forms-utils": "^1.3.5"
+		"infinum/eightshift-forms-utils": "dev-feature/test-libs"
```

Comment:
Will the feature branch be updated to a stable one before you release this?

Reply:
correct once everything is done